Changed order of images for the device database section (#43368)
home-assistant/home-assistant.github.io: source/_posts/2026-02-04-release-20262.markdown

Since we’re still in the early stages, the device database lives in {% my labs title="**Home Assistant Labs**" %}, where you can opt in to share anonymized information about the devices in your home.

<img src='/images/blog/2026-02/temp-sharing-your-devices-with-ohf-device-database.png' alt='Screenshot of the device analytics sharing option in Home Assistant Analytics.' class='no-shadow' />

> By introducing it in Labs, we have also introduced an additional section called **Device analytics** in {% my analytics title="**Home Assistant Analytics**" %}. If you opt in, you are of course able to opt out at any time.

Reviewer: ```suggestion
By introducing it in Home Assistant Labs, we have also added a new section called **Device analytics** to {% my analytics title="**Home Assistant Analytics**" %}. If you opt in, you are, of course, able to opt out at any time.
```
